pkgs/development/compilers/llvm/17/libclc/gnu-install-dirs.patch

```diff
@@ -0,0 +1,10 @@
+--- a/libclc.pc.in
++++ b/libclc.pc.in
+@@ -1,5 +1,5 @@
+-includedir=@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_INCLUDEDIR@
+-libexecdir=@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_DATADIR@/clc
++includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
++libexecdir=@CMAKE_INSTALL_FULL_DATADIR@/clc
```

Comment:
Was this patch accidentally deleted in the LLVM 15 removal commit and re-added in the LLVM 16 one?

Seems wrong that it's added and not moved from somewhere.

Reply:
Rather, it looks like the commit should delete it from the LLVM 16 tree too. Not sure how that got lost.